curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md

```diff
@@ -68,13 +72,17 @@ tests:
 
 ```html
 <style>
- img {
-   max-width: 100%;
-   display: block;
-   height: auto;
- }
+.responsive-img {
+  max-width: 100%;
+  height: auto;
+}
+
+img {
+  width: 600px;
+}
 </style>
 
+<img class="responsive-img" src="https://s3.amazonaws.com/freecodecamp/FCCStickerPack.jpg" alt="freeCodeCamp stickers set">
 <img src="https://s3.amazonaws.com/freecodecamp/FCCStickerPack.jpg" alt="freeCodeCamp stickers set">
```

Comment:
Do we want to maybe remove this line from the solution? I haven't tried this so not sure if it'll work without this line.

Reply:
Not sure why we would remove it. It's part of the seed code so it should stay in the solution, right?

Reply:
Yeah. Sounds fine. I missed it somehow.

Reply:
I am still unable to get this to work using a Microsoft Edge browser

Also, when working through the task it has no mention of using <code>display</code> nor any mention of applying the 'block' setting.